Soccar [UI]/src/nl/soccar/ui/input/Keyboard.java

```diff
@@ -58,6 +53,9 @@
         HANDBRAKE = new ArrayList<>();
         HANDBRAKE.add(KeyCode.SPACE);
     }
+    
+    // Stores the keys that are being pressed at any time.
+    private final List<KeyCode> PRESSED_KEYS = new ArrayList<>();

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "PRESSED_KEYS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS00116)